### DISCUSSION TITLE
T1:Farm_InitFAST:FWrap_Init:MeshMapCreate:CreateLoadMap_L2_to_L2:CreateMapping_Pro jectToLine2:Node 17 does not project onto any line2 element. Closest distance is 1.844 m.

### DISCUSSION THREAD

**Discussion by tatesnow** [OpenFAST/openfast#1007] [Q&A] [answered]
When I run the fastfarm program, the following problem occurs.
![image](https://user-images.githubusercontent.com/42032942/154638927-507f0517-9185-4ded-80c1-68ff486bf5ad.png)
Do you know the reason for this problem？
Here is my fastfarm input file
[TSinflow.txt](https://github.com/OpenFAST/openfast/files/8095032/TSinflow.txt)

**Comment by tatesnow** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/42032942/154641079-77b674a5-fb76-4cf5-a83b-efd8f272dd99.png)

**Reply by tatesnow** [reply]
![image](https://user-images.githubusercontent.com/42032942/154658050-a431888d-938a-4c49-b201-9819d25e8d27.png)
And why keep reporting this error?

**Reply by jjonkman** [reply]
Dear @tatesnow,

Just a few comments:

- It looks like you are sharing three different issues, but you've only shared one input file.  How are you setting up the model differently between the three different cases you are showing?
- Regarding the first error, it appears that there is an incompatibility between the polar grid you've defined in FAST.Farm and the aerodynamic nodes in AeroDyn.  Can you share your OpenFAST model input file(s) as well?
- Regarding the second error, this looks identical to the issue reported in #808, which was fixed via PR #769.  This fix is implemented in the `dev` branch, but has not yet been merged into `main`.
- Regarding the last error, it looks like an analysis node extends outside of the wind domain.  It looks like your wind domain extends up to 281 m, but an analysis node extends up to 345 m.  Is your wind turbine too large for your wind domain?

Best regards,

**Reply by tatesnow** [reply]
Dear @jjonkman,
Thank you so much for your quick reply!

I've put the input file for openfast below
It seems that there is a problem with the gridheight setting value. I would like to ask if there are any regulations for the setting of this value? I want to set a fixed yaw error for the first unit to observe the influence on the downstream wind turbine load
[test.zip](https://github.com/OpenFAST/openfast/files/8101134/test.zip)

Best regards,

**Comment by tatesnow** [comment]
Dear @jjonkman,
I simulated three wind turbines in series using fastfarm and could get the final result, but I analyzed the data and found that the power of the third wind turbine was higher than the second one and even close to the first wind turbine. But according to my experience, the third wind turbine should generate less power than the second wind turbine.
Do you know why?
The relevant input files and output are inside the zip file.
inputfile
[input.zip](https://github.com/OpenFAST/openfast/files/8102101/input.zip)
output
T1 wind turbine
![image](https://user-images.githubusercontent.com/42032942/154800235-44a51c9f-6693-4e76-a22d-1a18ad47cd00.png)
T2 wind turbine
![image](https://user-images.githubusercontent.com/42032942/154800239-d559c6f0-fb4c-4207-ae1c-7fed5e3e1340.png)
T3 wind turbine
![image](https://user-images.githubusercontent.com/42032942/154800259-b4063828-e216-450f-a77c-86e806b09812.png)
Best regards,

**Comment by tatesnow** [comment]
When performing fastfarm simulation, are there any requirements for the parameters of Tmax? Is this closely related to the incoming wind speed? For example, the average wind speed is 10 m/s, and the distance between the first and last wind turbines is 2000 meters, so the data for the first 2000/10=200 seconds is unstable and cannot be for analysis, is my understanding correct?

**Reply by jjonkman** [reply]
Dear @tatesnow,

I haven't ran your example, but from my quick look, it looks you are simulating with a mean wind speed of 6 m/s, with a wind turbine spacing of 8D = 1008 m.  Thus, it will take about 1008/6 = 168 s for the wake of T1 to impinge upon T2 and for the wake of T2 to impinge upon T3 and twice that for the wake of T1 to impinge about T3.  This long start-up transient from FAST.Farm should be neglected in subsequent post-processing.  With turbulent wind input, I would suggest running FAST.Farm for a minimum of 10 minutes after the start-up transients pass.

Best regards,

**Reply by tatesnow** [reply]
Dear @jjonkman,
Thank you very much! Based on your suggestion, I reran the program and now it works fine！
 Best regards,
